CHANGELOG.md

```diff
@@ -1,3 +1,60 @@
+# v2.0.0-rc.0
+
+_Note: this is a prerelease for Cosign 2.0! Feel free to try it out, but know there are many breaking changes from 1.0 and the prereleases may continue to change._
```

Comment:
Might be worth putting the breaking changes from the PRs! I think each of them that did have them have a note